### ISSUE TITLE
stranded_CpG_report folder is empty

### ISSUE THREAD

**Issue by liuyangzzu** [nf-core/methylseq#297]
### Description of the bug

Hi,

I am using latest pipeline to call methylation from RRBS, however, the `stranded_CpG_report` folder is still empty, even I specify the argument `--cytosine_report` to run nextflow pipeline.

Before I use the older version, there is no problem. However, I found there is a big change of arguments for newest version.

Would you please help me to figure out where I know the reason of the empty folder issues? Thank you!

Best,
Yang

### Command used and terminal output

```console
nextflow run nf-core/methylseq\
    -profile singularity \
    --input samplesheet.csv \
    --outdir results\
    --skip_deduplication --rrbs --cytosine_report\
    --fasta /projects/li-lab/reference/hg38/hg38.fasta \
    --max_time '3.day' --max_memory '40.GB' --max_cpus 12 \
    --bismark_index ${refGenome}
```

**Comment by liuyangzzu** [comment]
A minimum test also show same issue:

```
nextflow run nf-core/methylseq -profile test,singularity \
    --outdir results \
    --cytosine_report
```

Then check the folder:
```
ls results/bismark/methylation_calls/stranded_CpG_report
```

**Comment by sateeshperi** [comment]
is this issue resolved @njspix ?

**Comment by njspix** [comment]
The files are created and put in `results/bismark/methylation_calls/methylation_calls` - as far as I can tell the `stranded_CpG_report` directory is just never used for anything (methylation calls are put in the above mentioned directory whether they are stranded or not). Not sure why the 'stranded_CpG_report' output directory is still getting created...

**Comment by sateeshperi** [comment]
yup i haven't seen them come up in tests so, can remove for sure :+1:
